test/functional/terminal/ex_terminal_spec.lua

```diff
@@ -112,4 +112,46 @@ describe(':terminal (with fake shell)', function()
       eq(2, eval("1+1"))  -- Still alive?
   end)
 
+  describe('works with', function()
+    it('findfile()', function()
+      source('terminal')
+      eq(1, nvim('eval', 'bufname("%") =~# "^term://"'))
+      eq('scripts/shadacat.py', nvim('call_function',
+                                     'findfile', {'scripts/shadacat.py', '.'}))
+    end)
+
+    it(':find', function()
+      terminal_with_fake_shell()
+      wait()
```

Comment:
wait() shouldn't be needed, screen:expect() gathers updates until it sees a valid state.